components/members/main/MemberList/index.tsx

```diff
@@ -60,23 +82,30 @@ const MemberList: FC = () => {
         <StyledMain hasProfile={hasProfile}>
           {!hasProfile && <StyledDivider />}
           {!isMobile ? (
-            <StyledMemberRoleMenu value={filter} onSelect={onSelect} />
+            <StyledMemberRoleMenu value={filter} onSelect={handleSelect} />
           ) : (
-            <StyledMemberRoleDropdown value={filter} onSelect={onSelect} />
+            <StyledMemberRoleDropdown value={filter} onSelect={handleSelect} />
           )}
           {/* TODO(@jun): 로딩 추가 */}
           <StyledCardWrapper>
-            {profiles?.map((profile) => (
-              <Link key={profile.id} href={playgroundLink.memberDetail(profile.id)} passHref>
-                <MemberCard
-                  name={profile.name}
-                  part={profile.part}
-                  isActiveGeneration={profile.isActive}
-                  introduction={profile.introduction}
-                  image={profile.profileImage}
-                />
-              </Link>
+            {profiles?.map((profiles, index) => (
+              <React.Fragment key={index}>
+                {profiles.map((profile) => (
+                  <Link key={profile.id} href={playgroundLink.memberDetail(profile.id)} passHref>
+                    <a>
+                      <MemberCard
+                        name={profile.name}
+                        part={profile.part}
+                        isActiveGeneration={profile.isActive}
+                        introduction={profile.introduction}
+                        image={profile.profileImage}
+                      />
+                    </a>
+                  </Link>
```

Comment:
Next 13부터 Link 컴포넌트의 동작 방식 변경으로 Link 안에 a태그가 이제 없어야해! 이거만 더 수정해줘! (참고: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor)

```suggestion
                  <Link key={profile.id} href={playgroundLink.memberDetail(profile.id)} passHref>
                      <MemberCard
                        name={profile.name}
                        part={profile.part}
                        isActiveGeneration={profile.isActive}
                        introduction={profile.introduction}
                        image={profile.profileImage}
                      />
                  </Link>
```



![image](https://user-images.githubusercontent.com/36122585/209970474-64bc07f0-4f72-421c-91b3-2029eecd7686.png)

Reply:
오 이거 알고있었는데 우리 13이었구나? (알게 된 경로가 우리 레포였었나..?ㅎㅎ 가물가물 하넵,,) 오키 ㅎㅎ 고마워!

Reply:
59d8176 요기로 반영했어요 ㅎㅎ